workflow-templates/py-formatting.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check Python formatting
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install isort and black
+        run: pip install isort black
```

Comment:
black does change from time to time; do we want/need to pin this or just accept that there will be migrations?

Reply:
I think they promise stability over a year. I'd be inclined to just reran black for the minor change if something changes.